packages/matter-node.js/src/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @mfucci
```

Comment:
There can be only one CODEOWNERS file per repository.
Delete this one...

Reply:
missed that one to delete ...